Anchal_Hora/panel/WebApplicationPanel/Controllers/LoginController.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using WebApplicationPanel.DbModels;
+
+// For more information on enabling MVC for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace WebappAPItype.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+
+    public class LoginController : Controller
+    {
+        private panelContext _panelcontext;
+        private IConfiguration _config;
+        private panelContext _panelContext;
+
+        public LoginController(IConfiguration config, panelContext panelcontext)
+
+        {
+            _panelcontext = panelcontext;
+            _config = config;
+        }
+
+        [HttpPost]
+        public IActionResult Login([FromBody] WebApplicationPanel.DbModels.EmpInformation login)
```

Comment:
Remove unnecessary path (" WebApplicationPanel.DbModels") to access Models.

Reply:
Path removed

Reply:
WebApplicationPanel.DbModels.EmpInformation still in use